### DISCUSSION TITLE
Welcome to Terrascan Discussions!

### DISCUSSION THREAD

**Discussion by amirbenv** [accurics/terrascan#750] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Share how you are using Terrascan
  * Share how you **would like** to use Terrascan! Let us know what **new features** you'd like to see
  * Ask any questions you’re wondering about
  * Discuss your thoughts about IaC, GitOps, and Security Scanning
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by UK-NEO** [comment]
I am doing classification of LiDAR points. when I classified the point cloud, the water part usually created water polygons. Is there a way to automatically create this water polygon?

**Reply by gaurav-gogia** [reply]
Hey @UK-NEO 

Could you please share some more context? Terrascan is a SAST tool for Infrastructure as Code files like Terraform/Docker/K8s etc. I am unable to make the connection between your query and Terrascan.

**Reply by UK-NEO** [reply]
hello, thanks Gauray-Gogia,
I am a GIS engineer who acquires LiDAR data from aircraft and processes that data to create terrain models.
LiDAR data becomes point cloud data, and this point cloud data is classified as ground, building, road, and  vegetation etc.
At that time, most of the points of water are missing, Partially processed as a point cloud, it is done manually when classifying the points as water. TerraSolid's TerraScan is used for the classification process. I'm looking for a way to make automatic process.

**Reply by gaurav-gogia** [reply]
> TerraSolid's TerraScan is used for the classification process. I'm looking for a way to make automatic process.

It looks like you're talking about a different software here. [TerraSolid TerraScan](https://terrasolid.com/products/terrascan/) is different from [Tenable Terrascan](https://runterrascan.io/)

I am unable to find their page on GitHub. I do understand the reason for confusion though.

So, for clarification. This is a discussion page for Tenable's Terrascan. A security product for Infrastructure as Code files. It is not related to any GIS software out there.

**Reply by UK-NEO** [reply]
I appreciate your kind help. I was looking for the answer to my question in the wrong place.
I try to see if there is a forum discussing Terrasolid's TerraScan.
Thank you very much again.
